jetty-server/src/main/config/etc/jetty-acceptratelimit.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Call name="addBean">
+    <Arg>
+      <New class="org.eclipse.jetty.server.AcceptRateLimit">
+        <Arg name="maxRate" type="int"><Property name="jetty.acceptrate.limit" default="1000" /></Arg>
```

Comment:
This is against the naming convention for modules and their properties.

It's `jetty.<module_name>.<property>`.

So this module should be named `acceptratelimiter`, and the properties be `jetty.acceptratelimiter.maxConnections` and `jetty.acceptratelimiter.suspendPeriod`.

Both the current `limit` and `period` have zero meaning in their name (the limit of what ? the period of what ?)